lib/sentry_conn.py

```diff
@@ -0,0 +1,51 @@
+import requests
+
+
+class APIClient:
+    def __init__(self, base_url):
+        self.api_token = ''
+        self.project_slug = ''
+        if not base_url.endswith('/'):
+            base_url += '/'
+        self.__url = (
+                '{0}api/0/'
+            ).format(base_url)
+
+    def get(self, uri):
+        headers = {
+            'Authorization': f'Bearer {self.api_token}',
+            'Content-Type': 'application/json'
+        }
+
+        url = self.__url + uri
+        all_results = []
+
+        while url:
+            print(f"Fetching: {url}")
+            response = requests.get(url, headers=headers)
+            response.raise_for_status()
+
+            data = response.json()
+            if isinstance(data, list):
+                all_results.extend(data)
+                print(f"Received {len(all_results)} items")
+            else:
+                return data  # For non-list endpoints
+
+            link_header = response.headers.get("Link", "")
```

Comment:
I have handled pagination in sentry_conn.py. The "next" link is queried until there's no more items.